client/app/core/navigation.service.js

```diff
@@ -123,4 +130,10 @@ export function NavigationFactory (RBAC, Polling, POLLING_INTERVAL, $ngRedux, Co
       })
     })
   }
+
+  function setPermissions () {
+    menuItems[1].permissions = RBAC.has(RBAC.FEATURES.SERVICES.VIEW)
```

Comment:
Can we create constants for these array positions. I know its simple but it will make the code a little easier to read.

Reply:
The assignment is self informing, I don't so much agree with creating variables that server no other purpose than to increase verbosity .